product_docs/docs/epas/12/epas_rel_notes/19_epas12.1.2_rel_notes.mdx

```diff
@@ -191,5 +191,5 @@ The following components are included in the EDB Postgres Advanced Server v12 re
 -   pgBouncer 1.12.0
 -   pgPool-II & pgPool-IIExtensions 4.0.6
 -   MTK 53.0.0
--   EDBPlus 38.0.0
+-   EDBPlus 38.0.1
```

Comment:
Isn't this list what was tested for the initial release of EPAS v12? I don't think we need to update the version number here.